File: Java/백준/Bronze/28938. Конвейер/Конвейер.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(); int sum=0;
        for(int i=0; i<a; i++){
            sum+=in.nextInt();
        }
        if(sum==0){
            System.out.println("Stay");
        }
        else if(sum>0){
            System.out.println("Right");
        }
        else{
            System.out.println("Left");
        }
    }
}

File: C#/백준/Bronze/28938. Конвейер/Конвейер.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int sum=0;
            string[] aa = Console.ReadLine().Split();
            for (int i =0; i<a; i++){
                sum+=int.Parse(aa[i]);
            }
            if(sum==0){
                Console.Write("Stay");
            }
            else if(sum>0){
                Console.Write("Right");
            }
            else{
                Console.Write("Left");
            }
        }
    }
}